src/components/search/Search.tsx

```diff
@@ -21,11 +21,12 @@ import { SortingMenu } from './sorting/SortingMenu/SortingMenu'
 
 export const Search: FC = observer(() => {
     const { classes } = useStyles()
+    const inputRef = useRef<HTMLInputElement>(null)
```

Comment:
doesn't it break keyboard shortcuts functionality?

Reply:
I'm not sure, how do I check?

This basically achieves the same result as before, where we had `useRef` within the SearchViewStore. I've removed the useRef from the SearchViewStore because it was breaking the rules of react hooks, whenever I tried to wrap the `new SharedStore(user)` with `useMemo`. Other hooks cannot be wrapped with `useMemo`. And in general its a rule to not use hooks within `class`.

Please let me know how to check for the keyboard shortcut.

Reply:
here are the shortcuts help, `J` and `K` currently doesn't work but the rest should be fine
<img width="1677" alt="image" src="https://github.com/liquidinvestigations/hoover-ui/assets/9444169/8ff17ff2-e2ae-412f-b556-c9d2b7049e71">

Reply:
I checked it and it's working fine, the and the reason of keeping `ref` there instead of element returned by callback was the requirement of `ref` object as MUI TextField, so lets keep your changes